scripts/prepare-db-layer.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+##
+# This is expected to be invoked during the bundling of the database support Lambda layer.
+# The CDK supports only a single bundling command and so it is easier to execute all of the
+# necessary commands in a script rather than a series of `&&`-joined commands in a TypeScript
+# file.
+##
+
+export ASSET_INPUT_DIR="/asset-input"
+export ASSET_OUTPUT_DIR="/asset-output"
+
+## Transpile entities and migrations for TypeORM 
+prepare-orm-files() (
+    npm ci \
+        && npm run lerna bootstrap -- --ci \
+        && npm run build \
+        && cd "$ASSET_INPUT_DIR/build/packages/api" \
+        && cp --parents -aur orm/**/**.js "$ASSET_OUTPUT_DIR" \
```

Comment:
I understand the first double asterisk globbing operator.  What is the significance of the second?  Does it do something different than `*.js` ?

Reply:
Honestly, not sure. `orm/**.js` didn't catch the things under subdirectories. So `orm/**` should catch `entities` and `migrations` and then `orm/**/**.js` should catch all the JS files within each of those directories. Any suggestions on cleaning up this glob? I was just sorta going with the thing that worked first here :D

Reply:
Nah, this is good.  I was just curious.